File: omoda5-backup/backup/apps/media/java_src/com/chery/media/model/receiver/BootReceiver.java
package com.chery.media.model.receiver;

import android.content.BroadcastReceiver;
import android.content.ComponentName;
import android.content.Context;
import android.content.Intent;
import com.avn.tools.log.CarLog;
import com.avn.tools.log.PFLog;

/* loaded from: classes.dex */
public class BootReceiver extends BroadcastReceiver {
    private static final String TAG = CarLog.concatTag("MEDIA", BootReceiver.class);

    @Override // android.content.BroadcastReceiver
    public void onReceive(Context context, Intent intent) {
        PFLog.m25d(TAG, "onReceive " + intent);
        if (intent.getAction().equals("android.intent.action.BOOT_COMPLETED")) {
            PFLog.m25d(TAG, "to start UsbScannerManagerService");
            Intent intent2 = new Intent();
            intent2.setComponent(new ComponentName("com.chery.media", "com.chery.media.model.service.UsbScannerManagerService"));
            context.startService(intent2);
        }
    }
}

File: app/src/main/java/com/omoda5/launcher/service/BootReceiver.kt
package com.omoda5.launcher.service
import android.content.*
import android.util.Log
class BootReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        if (intent.action == Intent.ACTION_BOOT_COMPLETED || intent.action == "android.intent.action.QUICKBOOT_POWERON") {
            val serviceIntent = Intent(context, AdbBridgeService::class.java).apply { action = AdbBridgeService.ACTION_EXECUTE_SHELL }
            context.startForegroundService(serviceIntent)
        }
    }
}
